### ISSUE TITLE
"An Error Has Occured" after not using for 20 min or so

### ISSUE THREAD

**Issue by prestmango** [vincelwt/chatgpt-mac#84]
When the chat is not used for more than 20 minutes, and a new prompt is entered, it gives the error 
"An error occurred. If this issue persists please contact us through our help center at help.openai.com."

Every time, one needs to quit the app, or click "reload" and restart the whole thing.

**Comment by PaulRBerg** [comment]
Yup, can confirm that this happens on my end, too. I'm on macOS@13.1

**Comment by tomasz90** [comment]
Same here. "An error occurred. If this issue persists please contact us through our help center at help.openai.com."
Please take a look into it since this app is really neat and handy...

**Comment by Legitdove** [comment]
Just cmd-R whenever that happens to refresh. Should be fine after
